setup.cfg

```diff
@@ -79,7 +79,7 @@ install_requires =
     # Update CustomTTYColoredFormatter to remove
     colorlog>=4.0.2, <5.0
     configupdater>=3.1.1
-    connexion[flask]>=2.10.0
+    connexion[flask]>=2.10.0,<3.0
```

Comment:
Usual when we upper bound limit some package we add couple words about what the reason for it, and further steps to remove this limitation.
Could you also add this info?

Reply:
Yes. We need an explanation for that. Generally you should use constraints where you install airflow from scratch to achieve reproducible builds https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html  - so lack of "upper-binding" is EXPECTED in most dependencies. We need to have a good reason and explanation to upper-bind it.

Reply:
Or - even better - we need a fix for that.

Reply:
Sure, I'd just thought the git commit message was sufficient. I'll add some source comments

> we need a fix for that

Like adding a dynamic branch based on connexion version?

Reply:
No. Either commit explaining why we need it and what we are waiting for or containing a fix.

Reply:
Is the added comment sufficient? It would be difficult for me to be any more specific, given my limited familiarity with Airflow internals and connexion.

Reply:
Fix would be better if possible.